scripts/run_e2e_tests.py

```diff
@@ -585,6 +583,8 @@ def main(args=None):
             break
         # Prepare for rerun.
         cleanup()
+        for port in [OPPIA_SERVER_PORT, GOOGLE_APP_ENGINE_PORT]:
+            common.wait_for_port_to_be_closed(port)
```

Comment:
What if the port is not closed in the 60 sec, do we need to raise an exception?

Reply:
The rerun will fail because the server will still be running, so I didn't think we needed an extra exception raised here

Reply:
But the error message won't say the correct thing right? Should we stop the script here with the message `server is taking more than x sec to close.`?

Reply:
Done in 2ce96489d